src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrCodeflower.cs

```diff
@@ -48,8 +49,18 @@ public abstract class VmrCodeFlower : IVmrCodeFlower
     private readonly ILocalGitRepoFactory _localGitRepoFactory;
     private readonly IVersionDetailsParser _versionDetailsParser;
     private readonly IFileSystem _fileSystem;
+    private readonly ICommentCollector _commentCollector;
     private readonly ILogger<VmrCodeFlower> _logger;
 
+    private static readonly string CannotFlowAdditionalFlowsInPrMsg = string.Concat(
+        "The source repository has received code changes from an opposite flow. Any additional codeflows into ",
+        "this PR may potentially result in lost changes. Please continue with one of the following options: \n",
+        "1. Close or merge this PR and let the codeflow continue normally. \n",
+        "2. Close or merge this PR and receive the new codeflow immediately by running ",
+        "`darc trigger-subscription --id <subscriptionId>`. \n",
+        "3. Keep flowing changes into this PR anyway by force-triggering the subscription: ",
+        "`darc trigger-subscription --force --id <subscriptionId>`.");
```

Comment:
I think the new lines are missing in some of these place. Would it be better to just use a multi-line string?

```suggestion
    private static readonly string CannotFlowAdditionalFlowsInPrMsg =
        """
        The source repository has received code changes from an opposite flow. Any additional codeflows into
        this PR may potentially result in lost changes.
        
        Please continue with one of the following options:
        1. Close or merge this PR and let the codeflow continue normally
        2. Close or merge this PR and receive the new codeflow immediately by running  
            `darc trigger-subscription --id <subscriptionId>`
        3. Force-flow changes into this PR but at your own risk (some PR commits might be reverted):
            `darc trigger-subscription --force --id <subscriptionId>`
        """;
```

Reply:
Maybe it's more readable without all those `\n` and `",`?

Reply:
@adamzip my suggestion also had this part `at your own risk (some PR commits might be reverted)`. Maybe we should keep that?